.github/workflows/ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
       - run: cargo test
         env:
           RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg async_process_force_signal_backend
+        if: matrix.os != 'windows-latest'
```

Comment:
why?

Reply:
Windows now no longer supports the "signals" backend. As forcing the signal backend is not public API this is not a breaking change.